pkgs/stdenv/generic/setup.sh

```diff
@@ -1012,7 +1030,7 @@ buildPhase() {
         )
 
         echoCmd 'build flags' "${flagsArray[@]}"
-        make ${makefile:+-f $makefile} "${flagsArray[@]}"
+        performance-metrics "build" make ${makefile:+-f $makefile} "${flagsArray[@]}"
```

Comment:
Is this and the other calls to `performance-metrics` actually needed given that the duration of the phase is already tracked? The overhead of the phase is probably neglible.

Reply:
This was about collecting the real/user/system times. It's possible the times command might work. I need to play around with it some.